README.md

```diff
@@ -87,6 +87,22 @@ $ sudo ./build_all.sh
 **All builds runs must be done on the host machine that is building the images.** This is because we use
 the ansible_chroot connection type, which is not supported over ssh connections.
 
+the build artifacts will locate in folders as below( (which are defined  in "hosts" file respectively ):
```

Comment:
I think it might sound a little better to say "The build artifacts will be located in these directories below (which are defined in the `./hosts` file):"